mineinabyss-features/src/main/kotlin/com/mineinabyss/features/gondolas/GondolasHelpers.kt

```diff
@@ -11,10 +13,13 @@ enum class GondolaType() {
     NONE;
 }
 
-fun gondolaWarp(gondola: Gondola, player: Player, gondolaType: GondolaType) {
+fun gondolaWarp(gondola: Gondola, player: Player, gondolaType: GondolaType, gondolaId: String? = null) {
     val loc =
         if (gondolaType == GondolaType.LOWER) gondola.upperLoc else gondola.lowerLoc
```

Comment:
might be more "correct" to have this be a gondola.location(type) extension function in Gondola class?
as this seems like logic you might in theory use elsewhere, and thus it makes sense for it to exist on the object to easily fetch